### PR TITLE
To/From direction change for Azure ATP Sensor

### PR DESCRIPTION
**Pull request by PeterSelchDahl**
To/From direction change for Azure ATP Sensor

### CODE REVIEW

**Comment by PeterSelchDahl** [comment]
To/From direction change for Azure ATP Sensor

**Comment by PRMerger19** [comment]
@PeterSelchDahl : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: ATPDocs/atp-prerequisites.md [hunk 1]
```diff
@@ -248,21 +248,21 @@ The Azure ATP standalone sensor requires at least one Management adapter and at
...
-|LDAP|TCP and UDP|389|Domain controllers|Outbound|
-|Secure LDAP (LDAPS)|TCP|636|Domain controllers|Outbound|
-|LDAP to Global Catalog|TCP|3268|Domain controllers|Outbound|
-|LDAPS to Global Catalog|TCP|3269|Domain controllers|Outbound|
-|Kerberos|TCP and UDP|88|Domain controllers|Outbound|
+|LDAP|TCP and UDP|389|Domain controllers|Azure ATP Sensor|Outbound|
+|Secure LDAP (LDAPS)|TCP|636|Domain controllers|Azure ATP Sensor|Outbound|
+|LDAP to Global Catalog|TCP|3268|Domain controllers|Azure ATP Sensor|Outbound|
+|LDAPS to Global Catalog|TCP|3269|Domain controllers|Azure ATP Sensor|Outbound|
+|Kerberos|TCP and UDP|88|Domain controllers|Azure ATP Sensor|Outbound|
```
Comment (thesourcerer) [inline ATPDocs/atp-prerequisites.md:260]: opposite, those are all from the sensor to the DC

File: ATPDocs/atp-prerequisites.md [hunk 2]
```diff
@@ -248,21 +248,21 @@ The Azure ATP standalone sensor requires at least one Management adapter and at
...
 |Netlogon (SMB, CIFS, SAM-R)|TCP and UDP|445|All devices on network|Outbound|
```
Comment (thesourcerer) [inline ATPDocs/atp-prerequisites.md:261]: From Sensor missing

File: ATPDocs/atp-prerequisites.md [hunk 3]
```diff
@@ -248,21 +248,21 @@ The Azure ATP standalone sensor requires at least one Management adapter and at
...
-|Windows Time|UDP|123|Domain controllers|Outbound|
-|DNS|TCP and UDP|53|DNS Servers|Outbound|
-|Syslog (optional)|TCP/UDP|514, depending on configuration|SIEM Server|Inbound|
-|RADIUS|UDP|1813|RADIUS|Inbound|
+|Windows Time|UDP|123|Domain controllers|Azure ATP Sensor|Outbound|
+|DNS|TCP and UDP|53|DNS Servers|Azure ATP Sensor|Outbound|
```
Comment (thesourcerer) [inline ATPDocs/atp-prerequisites.md:263]: opposite, those are from the sensor

File: ATPDocs/atp-prerequisites.md [hunk 4]
```diff
@@ -248,21 +248,21 @@ The Azure ATP standalone sensor requires at least one Management adapter and at
...
+|Syslog (optional)|TCP/UDP|514, depending on configuration|Azure ATP Sensor|SIEM Server|Inbound|
```
Comment (thesourcerer) [inline ATPDocs/atp-prerequisites.md:264]: opposite, from the SIEM server to the sensor.

File: ATPDocs/atp-prerequisites.md (hunk 1 above)
Comment (shsagir) [reply ATPDocs/atp-prerequisites.md:260]: Updated

File: ATPDocs/atp-prerequisites.md (hunk 2 above)
Comment (shsagir) [reply ATPDocs/atp-prerequisites.md:261]: Updated

File: ATPDocs/atp-prerequisites.md (hunk 3 above)
Comment (shsagir) [reply ATPDocs/atp-prerequisites.md:263]: Updated

File: ATPDocs/atp-prerequisites.md (hunk 4 above)
Comment (shsagir) [reply ATPDocs/atp-prerequisites.md:264]: Updated

**Comment by shsagir** [comment]
@PeterSelchDahl  Thanks for your feedback. We will be submitting a documentation change based on your suggestion in a separate PR, and the updated article should be live this week. Please let us know if there's anything more we can do for you. To follow up on this issue, leave us a comment with an @mention.

**Pull request closed without merging**